sqlglot/expressions.py

```diff
@@ -3237,13 +3234,28 @@ def to_table(sql_path, **kwargs):
         return sql_path
     if not isinstance(sql_path, str):
         raise ValueError(f"Invalid type provided for a table: {type(sql_path)}")
-    table_parts = sql_path.split(".")
-    catalog, db, table_name = [
-        to_identifier(x) if x is not None else x for x in [None] * (3 - len(table_parts)) + table_parts
-    ]
+
+    catalog, db, table_name = [to_identifier(x) if x is not None else x for x in split_num_words(sql_path, ".", 3)]
     return Table(this=table_name, db=db, catalog=catalog, **kwargs)
 
 
+def to_column(sql_path: str, **kwargs) -> Column:
+    """
+    Create a column from a `[schema].[table]` sql path. Schema is optional.
+
+    If a column is passed in then that column is returned.
+
+    Args:
+        sql_path: `[schema].[table]` string
```

Comment:
it should be [table].[column]

Reply:
this is similar to def column but i guess it takes a path? i wonder if def column and def table could be merged with to_column and to_table so we don't need both